packages/root-config/src/applications.js

```diff
@@ -1,8 +1,19 @@
 const PluginImportPathPre = "zuri-plugin-";
 const PluginAppPathPre = "plugin-";
 
+const textAlign = () => {
```

Comment:
isn't this function not already implemented?

Reply:
It has been removed from the other file

Reply:
i have removed the function completely please review